### PR TITLE
Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by fabiand**

### CODE REVIEW

**Comment by fabiand** [comment]
@mazzystr please review

File: MAINTAINERS.md [hunk 1]
```diff
@@ -10,4 +10,6 @@ The current Maintainers Group for the Kubevirt Project consists of:
 | [Vasiliy Ulyanov](https://github.com/vasiliy-ul) | SuSE | |
 | [Chris Calligari](https://github.com/mazzystr) | Red Hat | |
 
+This list must be kept in sync with the [CNCF Project Maintainers list](https://github.com/cncf/foundation/blob/master/project-maintainers.csv).
```
Comment (jean-edouard) [inline MAINTAINERS.md:13]: Just a thought, having the list in the same order in both places would help with sync

**Comment by mazzystr** [comment]
@fabiand Please amend your commit and add commit signature

File: MAINTAINERS.md (hunk 1 above)
Comment (mazzystr) [reply MAINTAINERS.md:13]: CNCF has automation that sync's the csv

**Comment by mazzystr** [comment]
@fabiand I signed your commit for you :)

**Pull request merged**